pkg/asset/cluster/cluster.go

```diff
@@ -70,7 +71,13 @@ func (c *Cluster) Generate(parents asset.Parents) (err error) {
 	if err != nil {
 		return errors.Wrap(err, "failed to create temp dir for terraform execution")
 	}
-	defer os.RemoveAll(tmpDir)
+
+	tfKeep := strings.ToLower(os.Getenv("OPENSHIFT_INSTALL_KEEP_TERRAFORM"))
```

Comment:
Which terraform files are you looking to keep?

Reply:
This particular commit is only for CI debugging. When this is ready to be merged, this commit will be removed.